Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -186,5 +186,34 @@ public ActionResult<IEnumerable<TeamInviteViewModel>> GetTeamInvites()
             var username = AuthUtils.GetUsername(User);
             return Ok(_teamService.GetTeamInvites(username));
         }
+
+        [HttpPatch]
+        [Route("invites/{teamID}/status")]
+        public async Task<ActionResult<TeamInviteViewModel>> AcceptTeamInvite(int teamID, [FromBody] ParticipantTeamUploadViewModel acceptedInvite)
+        {
+            var username = AuthUtils.GetUsername(User);
+            var invite = _teamService.GetTeamInvite(teamID, username);
+            if (invite is null)
+                return Conflict("You were not invited by this team.");
```

Comment:
I'm not sure that 409 conflict is exactly the correct response for this case. Conflict responses are used specifically when a resource state is inconsistent with the requested replacement, usually on a PUT request. If an invite is not found, you should be returning 404 not found.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

Reply:
This is a bigger conversation as I see that there are other places in RecIM where this Conflict status is returned. Up to you whether you agree and want to fix it now or leave it for later discussion. I'm probably going to bring it up with Amos.